test/ERC721UserMintable.test.js

```diff
@@ -1298,4 +1375,146 @@ describe('SDK', () => {
       expect(contractFactoryMock).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('setContractURI', () => {
+    it('[SetContractURI] - should return an Error if contract is not deployed', () => {
+      eRC721UserMintable = new ERC721UserMintable(signer);
+
+      expect(() =>
+        eRC721UserMintable.setContractURI({
+          contractURI:
+            'https://www.cryptotimes.io/wp-content/uploads/2022/03/BAYC-835-Website-800x500.jpg',
+        }),
+      ).rejects.toThrow(
+        errorLogger({
+          location: ERROR_LOG.location.BaseERC721_setContractURI,
+          message: ERROR_LOG.message.contract_not_deployed_or_loaded,
+        }),
+      );
+    });
+
+    it('[SetContractURI] - should console.warn if contractURI is not a link ', async () => {
+      eRC721UserMintable = new ERC721UserMintable(signer);
+      const logSpy = jest.spyOn(console, 'warn');
+
+      await eRC721UserMintable.deploy({
+        name: 'name',
+        symbol: 'symbol',
+        baseURI: 'URI',
+        contractURI: 'contractURI',
+        maxSupply: 10,
+        price: ethers.utils.parseEther(1),
+        maxTokenRequest: 1,
+      });
+      await eRC721UserMintable.setContractURI({ contractURI: 'URI' });
+
+      expect(logSpy).toHaveBeenCalledWith('WARNING: The ContractURI "URI" is not a link.');
+      expect(logSpy).toHaveBeenCalledWith(
+        'WARNING: ContractURI should be a public link to a valid JSON metadata file',
+      );
+    });
+
+    it('[SetContractURI] - should return an Error if the contractURI is empty', () => {
+      eRC721UserMintable = new ERC721UserMintable(signer);
+
+      const uri = async () => {
+        await eRC721UserMintable.deploy({
+          name: 'name',
+          symbol: 'symbol',
+          baseURI: 'URI',
+          contractURI: 'contractURI',
```

Comment:
based on the test name ("[SetContractURI] - should return an Error if the contractURI is empty"), it should be empty here, no?